src/configure/configure.ts

```diff
@@ -116,6 +116,69 @@ class Orchestrator {
         }
     }
 
+    private async getAzureResource() {
+        var azureResourceSelector = ResourceSelectorFactory.getAzureResourceSelector(this.inputs.pipelineConfiguration);
+        var resource = null;
+        try {
+            resource = await azureResourceSelector.getAzureResource(this.inputs);
+            return resource;
+        }
+        catch (err) {
+            if (err.message === Messages.setupAlreadyConfigured) {
+                this.continueOrchestration = false;
+                return;
+            }
+            else {
+                throw err;
+            }
+        }
+    }
+
+    private async selectTemplate(resource: GenericResource, inputs: WizardInputs) {
+        switch (resource.type) {
+            case TargetResourceType.AKS:
+                let templateParameterHelper = new TemplateParameterHelper();
+                let template = this.inputs.potentialTemplates[0] as LocalPipelineTemplate;
+                inputs.pipelineConfiguration.template = this.inputs.potentialTemplates[0];
+                await templateParameterHelper.setParameters(template.parameters, inputs);
+                break;
+
+            case TargetResourceType.WebApp:
+                var shortlistedTemplates = [];
+                shortlistedTemplates = this.inputs.potentialTemplates.filter((template) => template.targetKind === resource.kind);
```

Comment:
I did not see code to merge windows and linux templates, then so we need to do this filtering here?

Reply:
The potentialTemplates array is having merged templates = linux, windows and both local, remote. As this stage, we are deciding the template based on what resource did the user select(TargetKind) and what all were supported in the template service(TemplateType).

Reply:
When I say merged for windows and linux, it means that we change the template label to hide windows and linux words from there.
This merge is different from merging of local and remote templates. May be we should call it something different to be clear. morphed templates??

Reply:
Since discussed on call, resolving it